tasks/st1096_db_commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+declare PT_command
+declare PT_database
+command=$PT_command
+database=$PT_database
+
+if [ -e "/etc/sysconfig/pe-puppetserver" ] || [ -e "/etc/default/pe-puppetserver" ] || [ -e "/etc/default/puppetserver" ] || [ -e "/etc/sysconfig/puppetserver" ] # Test to confirm this is a Puppetserver
+then
+  echo "Primary Server or Compiler node detected"   #Log Line to StdOut for the Console
+
+case $command in
```

Comment:
It would improve readability to store the appropriate query in each step of the `case` statement and run the query afterwards.  That way there is less copy/paste.

Reply:
Added a new parameter to assign the query and executed in the end. We can further add another variable to store the repeatable part of the whole query but I think it doesn't matter too much at them moment given the number of queries included.